qa/tasks/ceph_manager.py

```diff
@@ -1177,7 +1177,7 @@ def run_ceph_w(self):
                   "-w"],
             wait=False, stdout=StringIO(), stdin=run.PIPE)
 
-    def flush_pg_stats(self, osds, no_wait=None, wait_for_mon=3*5):
+    def flush_pg_stats(self, osds, no_wait=None, wait_for_mon=300):
```

Comment:
could you update the comment at https://github.com/ceph/ceph/pull/16322/files#diff-4ea60646bd50b2d9d5b311878c1881d6R1191 accordingly?